### PR TITLE
Added serverless

### PR DESCRIPTION
**Pull request by shawju**
## what
* Added [serverless](https://github.com/serverless/serverless)

## why
* Need to use serverless to create Lambdas

### CODE REVIEW

**Review by bstascavage — DISMISSED** [review]

**Comment by bstascavage** [comment]
@shawju make sure you have your settings correct: https://github.com/saic-oss/anvil/pull/23#issuecomment-845970024

**Review by RothAndrew — CHANGES_REQUESTED** [review]
- Delete src/.DS_Store

Other than that it LGTM

**Comment by RothAndrew** [comment]
I'm reviewing from a compliance perspective only. Should get another approval from anvil team before merging

File: src/docker/Dockerfile [hunk 1]
```diff
@@ -85,7 +86,7 @@ RUN wget -O /root/task.tar.gz "https://github.com/go-task/task/releases/download
 # Install shellcheck. Get versions from https://github.com/koalaman/shellcheck/releases
 ARG SHELLCHECK_VERSION="0.7.1"
 ENV SHELLCHECK_VERSION=${SHELLCHECK_VERSION}
-# hadolint ignore=DL3003
+# hadolint ignore=DL3003,DL3047
```
Comment (bstascavage) [inline src/docker/Dockerfile]: I don't think we want this ignore here.  We should update `wget` to use the `--progress` flag.

See [here](https://github-wiki-see.page/m/hadolint/hadolint/wiki/DL3047)

File: src/docker/Dockerfile [hunk 2]
```diff
@@ -183,20 +195,23 @@ RUN asdf plugin add python \
 # NOTE: PIPENV_VERSION is an env var that is used by pipenv, so it can't be used here. ref: https://github.com/pypa/pipenv/issues/3633#issuecomment-478250721
 ARG PIP_ENV_VERSION="2020.11.15"
 ENV PIP_ENV_VERSION=${PIP_ENV_VERSION}
+# hadolint ignore=DL3042
```
Comment (bstascavage) [inline src/docker/Dockerfile]: I don't think we want this ignore here.

See [here](https://github-wiki-see.page/m/hadolint/hadolint/wiki/DL3042)

File: src/docker/Dockerfile [hunk 3]
```diff
@@ -118,33 +121,42 @@ RUN wget -O /root/fossa.tar.gz "https://github.com/fossas/fossa-cli/releases/dow
...
+# This is caused by Red Hat's packaging and there isn't much we can do about it.
+ARG SKOPEO_BINARY_VERSION="1.3.2"
+ENV SKOPEO_BINARY_VERSION=${SKOPEO_BINARY_VERSION}
+
 RUN dnf -y install skopeo-${SKOPEO_VERSION} \
   && dnf clean all \
   && rm -rf /var/cache/yum/ \
   && rm -rf /var/tmp/* /tmp/* /var/tmp/.???* /tmp/.???*
 
+# hadolint ignore=DL3059
```
Comment (bstascavage) [inline src/docker/Dockerfile:155]: What does this do?

**Review by bstascavage — CHANGES_REQUESTED** [review]
We shouldn't be adding `hadolint` ignores unless we explicitly need the acception.

**Review by bstascavage — APPROVED** [review]
LGTM

File: spec/skopeo_spec.sh [hunk 4]
```diff
@@ -1,13 +1,13 @@
...
+printVersion() {
+  skopeo -v | grep "${SKOPEO_BINARY_VERSION}"
+}
 
-  It "validates tool is installed by checking version"
-    When call printVersion
-    The output should include "${SKOPEO_VERSION}"
-    The status should eq 0
-  End
+It "validates tool is installed by checking version"
```
Comment (bstascavage) [inline spec/skopeo_spec.sh]: One last thing

This file needs to be indented like the other `shellspec` files

**Review by bstascavage — CHANGES_REQUESTED** [review]

**Review by RothAndrew — APPROVED** [review]
Approving from compliance perspective only. Get a 2nd from @saic-oss/anvil

**Review by bstascavage — APPROVED** [review]
LGTM

**Pull request merged**